### ISSUE TITLE
Version 7.0.3 doesn't work with serverside rendering

### ISSUE THREAD

**Issue by zdila** [andreypopp/react-textarea-autosize#213]
Version 7.0.3 doesn't work with serverside rendering. Version 6.1.0 works.

```
/home/joe/my-project/node_modules/react-textarea-autosize/dist/react-textarea-autosize.js:5
}(this, (function (exports,React) { 'use strict';
 ^
ReferenceError: document is not defined
    at /home/joe/my-project/node_modules/react-textarea-autosize/dist/react-textarea-autosize.js:840:14
    at /home/joe/my-project/node_modules/react-textarea-autosize/dist/react-textarea-autosize.js:2:66
    at Object.<anonymous> (/home/joe/my-project/node_modules/react-textarea-autosize/dist/react-textarea-autosize.js:5:2)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
```

**Comment by Andarist** [comment]
Why do you use UMD bundle in your SSR setup? `require('react-textarea-autosize')` should resolve to CJS file and I'm pretty sure it would work for you. 

That being said I guess I need to stop treating UMD as targeting browser as it should be *universal* after all.

**Comment by zdila** [comment]
I was importing it with `import Textarea from 'react-textarea-autosize/dist/react-textarea-autosize.js';` for some reason. Chaging it to `import Textarea from 'react-textarea-autosize';` fixes it. Thanks and sorry for bothering :-)

**Comment by Andarist** [comment]
Cool, that you could fix it. I've fixed UMD bundle too and published it as 7.0.4
